core/blockchain_af.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 
 	"github.com/ethereum/go-ethereum/common"
+	emath "github.com/ethereum/go-ethereum/common/math"
```

Comment:
Is there a nicer way to handle this? I notice only one function is needed from common/math

```
// BigPow returns a ** b as a big integer.
func BigPow(a, b int64) *big.Int {
	r := big.NewInt(a)
	return r.Exp(r, big.NewInt(b), nil)
}
```

Reply:
Indeed -- done with https://github.com/etclabscore/core-geth/pull/181/commits/36bb18e8cb5a636fd16fb5be0ebdb2f0f4ebad1c